docs/tutorials/services/color-detection-scuttle.md

```diff
@@ -6,7 +6,7 @@ type: "docs"
 description: "Instructions for detecting and following a colored object with a SCUTTLE Robot on Viam software."
 webmSrc: "/tutorials/videos/scuttle-colordetection-preview.webm"
 mp4Src: "/tutorials/videos/scuttle-colordetection-preview.mp4"
-videoAlt: "Detecting color with a Scuttle Robot"
+videoAlt: "Detecting color with a SCUTTLE Robot"
```

Comment:
```suggestion
videoAlt: "Detecting color with a Scuttle Robot"
```
I think If you write it as SCUTTLE it will be read as S C U T T L E. So here capitalizing is not a good thing

Reply:
Ohh I thought it was the opposite, and that's why we spell GPIO G P I O in alt text, because it reads it?